docs/sphinx/source/api/Installation.md

```diff
@@ -37,7 +37,9 @@ cd MDF
 ### 5) Install the package
 ```
 pip install .
+
 ```
+Hello world
```

Comment:
I don't think we want this committed---it was just a test to see if your `make html` was working.

Reply:
@kusanele yes, this change shouldn't have been committed. Just change the file back locally and commit that.